src/lib/apiDocs/isGroupValid.md

```diff
@@ -1,4 +1,4 @@
 ### `this.isGroupValid(groupName)`
 
-Returns `true` or `false` depending on if all the fields in form group `groupName` are valid.
+Returns `true` if all fields in the form group `groupName` are valid, or `false` if invalid.
```

Comment:
This is not *quite* true; only *one* field needs to be `invalid` for this to return `false`